x/globalfee/migrations/v2/migration.go

```diff
@@ -0,0 +1,37 @@
+package v2
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/cosmos/gaia/v10/x/globalfee/types"
+)
+
+// MigrateStore performs in-place params migrations of
+// BypassMinFeeMsgTypes and MaxTotalBypassMinFeeMsgGasUsage
+// from app.toml to globalfee params.
+// The migration includes:
+// Add bypass-min-fee-msg-types params that are set
+// ["/ibc.core.channel.v1.MsgRecvPacket",
+// "/ibc.core.channel.v1.MsgAcknowledgement",
+// "/ibc.core.client.v1.MsgUpdateClient",
+// "/ibc.core.channel.v1.MsgTimeout",
+// "/ibc.core.channel.v1.MsgTimeoutOnClose"] as default and
+// add MaxTotalBypassMinFeeMsgGasUsage that is set 1_000_000 as default.
+func MigrateStore(ctx sdk.Context, globalfeeSubspace paramtypes.Subspace) error {
+	var oldGlobalMinGasPrices sdk.DecCoins
+	globalfeeSubspace.Get(ctx, types.ParamStoreKeyMinGasPrices, &oldGlobalMinGasPrices)
+	defaultParams := types.DefaultParams()
+	params := types.Params{
+		MinimumGasPrices:                oldGlobalMinGasPrices,
+		BypassMinFeeMsgTypes:            defaultParams.BypassMinFeeMsgTypes,
+		MaxTotalBypassMinFeeMsgGasUsage: defaultParams.MaxTotalBypassMinFeeMsgGasUsage,
+	}
+
+	if !globalfeeSubspace.HasKeyTable() {
+		globalfeeSubspace.WithKeyTable(types.ParamKeyTable())
```

Comment:
@yaruwangway @MSalopek I don't think this line does anything. Correct me if I'm wrong. `WithKeyTable` seems to return a modified subspace that is thrown out here

Reply:
i think it should already registered keytable, this is not necessary here.  but it does not harm to recheck. keytable regissters the kv, and the validation methods.

Reply:
We should use the returned value tho right?
```suggestion
		newSubspace := globalfeeSubspace.WithKeyTable(types.ParamKeyTable())
```

Reply:
fixed in this PR, thank you! https://github.com/cosmos/gaia/pull/2524